include/util.cpp

```diff
@@ -0,0 +1,37 @@
+#include "util.h"
+
+void ShowDebugInfo(PaStream* stream)
```

Comment:
This is new; I added some more debug info here. Pretty much whatever portaudio will give me. Hopefully this will end up being useful in the future.